code/modules/surgery/organs/tongue.dm

```diff
@@ -137,7 +137,7 @@
 /obj/item/organ/tongue/abductor/handle_speech(datum/source, list/speech_args)
```

Comment:
```suggestion
/obj/item/organ/tongue/abductor/handle_speech(mob/user, list/speech_args)
```
the signal is sent from /mob so its always a mob

Reply:
All other tongues do it with the typecast, and changing them all is out of scope for this PR.